index.d.ts

```diff
@@ -0,0 +1,39 @@
+declare module 'envvar' {
+  /**
+   * Retrieves a given environment variable as a string.
+   *
+   * @throws if the environment variable is not set and a default is not
+   *         specified.
+   * @throws if the value of the environment variable is not a string.
+   */
+  export function string(name: string, defaultValue?: string): string;
+
+  /**
+   * Retrieves a given environment variable as a number.
+   *
+   * @throws if the environment variable is not set and a default is not
+   *         specified.
+   * @throws if the value of the environment variable is not a number.
```

Comment:
We should state _does not represent a number_ rather than _is not a number_ since the value of the environment variable will never actually be a number.